[docs] Document new maxSessions config option (#151268)
elastic/kibana: docs/settings/security-settings.asciidoc

+
TIP: Use a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h', '7d', '1w').

xpack.security.session.сoncurrentSessions.maxSessions::

> Set the maximum number of users (sessions) who are allowed to be logged into {kib} at the same time.

Reviewer: The wording here feels ambiguous. What do you think about something like this? 

```suggestion
Set the maximum number of sessions each user is allowed to have active at any given time.
```
